opennms-base-assembly/src/main/filtered/etc/datacollection/juniper.xml

```diff
@@ -44,6 +52,12 @@
         <mibObj oid=".1.3.6.1.4.1.4874.2.2.2.1.9.4.1.4" instance="juniSystemTempIndex" alias="juniSTPhysicalIndex" type="string" />
       </group>
 
+      <group name="juniper-srx-router" ifType="all">
+        <mibObj oid=".1.3.6.1.4.1.2636.3.39.1.12.1.1.1.11" instance="jnxJsSPUMonitoringObjectsTable" alias="jnxJsSPUMonitoringNodeDescr"          type="string" />
```

Comment:
The restriction of the 19 characters for the DS name is for numeric variables only (i.e. integer, counter, gauge, etc. or whatever numeric metric that must be stored inside the RRD file). The MibObj defined as string can be as long as you wish.